rest-api/resources/src/main/resources/openapi/jobExecution/jobExecution.yaml

```diff
@@ -27,6 +27,27 @@ components:
       allOf:
         - $ref: '../openapi.yaml#/components/schemas/kapuaUpdatableEntity'
         - type: object
+          properties:
+            startedOn:
+              description: The moment when the job execution started
+              type: string
+              format: 'date-time'
+            endedOn:
+              description: The moment when the job execution ended
+              type: string
+              format: 'date-time'
+            log:
+              description: logs for the job execution
+              type: string
+            jobId:
+              description: job id for this job execution
+              $ref: '../openapi.yaml#/components/schemas/kapuaId'
+            targetIds:
```

Comment:
This is an Array of `KapuaId` not an Array of `JobTarget`s

Reply:
fixed